test/functional/JIT_Test/src/jit/test/stringconcat/StringConcatTest.java

```diff
@@ -80,6 +80,16 @@ public void testNullStr_Middle(){
 	private String buildNullStr_Middle(){
 		return Integer.toString(-5) + " is bigger than " + Integer.toString(1) + null + " to a less than sane person";
 	}
+
+	private String concatNonAsciiChar(String s){
+		return new StringBuilder().append(s).append('\u205e').toString();
+	}
```

Comment:
Does this run with a variation that ensures the JIT will compile this method?

Reply:
The test runs as follows https://github.com/eclipse/openj9/blob/master/test/functional/JIT_Test/playlist.xml#L52

Reply:
`-Xjit:count=1,disableAsyncCompilation,verbose,vlog=$(Q)$(REPORTDIR)$(D)jitv.log$(Q)`

Reply:
Yeah, it had the `count=1,disableAsyncCompilation` which we need. However I forgot about adding `-XX:+CompactStrings`. I've added a new variation so now we will have that option as well: 4c32563

Reply:
We do use the `s` when we invoke `append` for the first time:
```
return new StringBuilder().append(s)...
```

Reply:
RIght sorry I missed the use of `s` that is fine.